Makefile

```diff
@@ -20,10 +20,10 @@ WASM_PACK ?= ~/.cargo/bin/wasm-pack
 endif 
 
 .PHONY: install
-install: node_modules/.yarn-integrity $(CARGO) $(WASM_PACK) ## Install dependencies
+install: node_modules/.npm-integrity $(CARGO) $(WASM_PACK) ## Install dependencies
```

Comment:
If we end up replacing Yarn with NPM, I think we should use `node_modules/.package-lock.json` as the install flag instead, since that's what NPM uses internally. We can also likely remove the `type`/`touch` lines below this as NPM (unlike Yarn) seems to always write to this file. Let me know if you want me to add a commit here.